hazelcast/src/test/java/com/hazelcast/scheduledexecutor/ScheduledExecutorServiceTestSupport.java

```diff
@@ -141,30 +141,34 @@ public void setHazelcastInstance(HazelcastInstance hazelcastInstance) {
 
     static class ICountdownLatchMapIncrementCallableTask implements Runnable, Serializable, HazelcastInstanceAware {
 
-        final String runLatchName;
+        final String startedLatch;
+        final String finishedLatch;
         final String runEntryCounterName;
         final String mapName;
 
         transient HazelcastInstance instance;
 
-        ICountdownLatchMapIncrementCallableTask(String mapName, String runEntryCounterName, String runLatchName) {
+        ICountdownLatchMapIncrementCallableTask(String mapName, String runEntryCounterName,
+                                                String startedLatch, String finishedLatch) {
             this.mapName = mapName;
             this.runEntryCounterName = runEntryCounterName;
-            this.runLatchName = runLatchName;
+            this.startedLatch = startedLatch;
+            this.finishedLatch = finishedLatch;
         }
 
         @Override
         public void run() {
             instance.getAtomicLong(runEntryCounterName).incrementAndGet();
+            instance.getCountDownLatch(startedLatch).countDown();
 
             IMap<String, Integer> map = instance.getMap(mapName);
-            for (int i = 0; i < 100000; i++) {
+            for (int i = 0; i < 10000; i++) {
```

Comment:
Same value in at least 3 places, could be replaced with a constant in `ScheduledExecutorServiceTestSupport` to ensure that it's the same everywhere.